pkg/validation/internal/community.go

```diff
@@ -256,10 +252,13 @@ func validateImageFile(checks CommunityOperatorChecks, deprecatedAPImsg string)
 						}
 
 						if verParsed.GE(semVerOCPV1beta1Unsupported) {
-							checks.errs = append(checks.errs, fmt.Errorf("this bundle is %s. Migrate the API(s) "+
-								"for %s or use the OCP labels for compatible version(s). (e.g. LABEL %s='=v4.8')",
+							checks.errs = append(checks.errs, fmt.Errorf("this bundle is %s. Migrate the API(s) for "+
```

Comment:
nit: I know there's precedent in this file, but I would like to stop using string concatenation as a way of making literals more readable for engineers -- almost all text editors have line-wrapping as a standard.